crates/bls-snark-sys/src/signatures.rs

```diff
@@ -44,17 +45,19 @@ pub extern "C" fn sign_message(
     in_extra_data: *const u8,
     in_extra_data_len: c_int,
     should_use_composite: bool,
+    should_use_cip22: bool,
```

Comment:
Nitpick, but it might be better to have a "hash mode" enum instead of two booleans? That has direct hash, composite hash, and composite hash with cip22

Reply:
I considered that, but didn't want to make the FFI interface with complicated types